spec/cases/each_with_ar_sqlite.rb

```diff
@@ -29,11 +27,19 @@ class User < ActiveRecord::Base
 print "Parent: "
 puts User.first.name
 
+# https://www.sqlite.org/faq.html
+# Under Unix, you should not carry an open SQLite database across a fork() system call into the child process.
+ActiveRecord::Base.connection.disconnect!
+
 print "Parallel (#{in_worker_type}): "
 Parallel.each([1], in_worker_type => 1) do
+  ActiveRecord::Base.establish_connection(params)
   puts User.all.map(&:name).join
 end
 
+ActiveRecord::Base.establish_connection(params)
 print "\nParent: "
 puts User.first.name
 
+# Delete the sqlite3 file.  :memory: was neat, but once you disconnect, it's just gone.
+`rm parallel_with_ar_test.sqlite3`
```

Comment:
no newline at end of file -> configure your editor to always add it for peace of mind